<ui version="4.0" >
 <class>NumberWidget</class>
 <widget class="QWidget" name="NumberWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="grouNumbering" >
     <property name="title" >
      <string>&amp;Numbering</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="labelStart" >
          <property name="text" >
           <string>Start &amp;index:</string>
          </property>
          <property name="buddy" >
           <cstring>spinStart</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="spinStart" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;Step by:</string>
          </property>
          <property name="buddy" >
           <cstring>spinStep</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="spinStep" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkReset" >
        <property name="text" >
         <string>&amp;Reset counter for every directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSkip" >
     <property name="title" >
      <string>&amp;Skip Numbers</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QListWidget" name="listSkip" />
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QPushButton" name="buttonAdd" >
            <property name="text" >
             <string>&amp;Add Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRemove" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>&amp;Remove Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinNumber" />
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
